mmdet3d/models/roi_heads/bbox_heads/parta2_bbox_head.py

```diff
@@ -220,14 +222,14 @@ def __init__(self,
 
         self.conv_reg = nn.Sequential(*reg_layers)
 
-        self.init_weights()
+        if init_cfg is None:
+            self.init_cfg = dict(
+                type='Xavier',
+                layer=['Conv2d', 'Conv1d'],
+                distribution='uniform')
 
     def init_weights(self):
-        """Initialize weights of the bbox head."""
-        for m in self.modules():
-            if isinstance(m, (nn.Conv2d, nn.Conv1d)):
-                xavier_init(m, distribution='uniform')
-
+        super().init_weights()
         normal_init(self.conv_reg[-1].conv, mean=0, std=0.001)
```

Comment:
Is this hard-coded init inevitable?

Reply:
yes,  now mmcv does not support this kind of ops